docs/README.Windows.md

```diff
@@ -261,3 +263,26 @@ UWP builds generate `msix`, `appxsym` and `cer` files, located inside directorie
 
 
 **[back to top](#table-of-contents)** | **[back to section top](#6-build-kodi-manually)**
+
+## 7. Build binary add-ons
+You can find a complete list of available binary add-ons **[here](https://github.com/xbmc/repo-binary-addons)**.
+
+### 7.1. In-tree building of binary add-ons
+Change to the 64bit build directory (**recommended**) or choose the appropriate architecture:
```

Comment:
64bit isnt really "recommended" as such. I think this would be better worded like

Change to the architecture platform you wish to build for (examples use x64)

Reply:
done